src/main/java/uk/gov/companieshouse/model/dto/payment/PaymentPatchRequest.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.companieshouse.model.dto.payment;
+
+
+import uk.gov.companieshouse.model.enums.PaymentStatus;
+
+import java.sql.Timestamp;
+
+public class PaymentPatchRequest {
+    private PaymentStatus status;
+
+    private String paymentReference;
```

Comment:
I think you need a `@JsonProperty('payment_reference')` due to the snake_case mapping. Otherwise swagger won't generate accurately.